### DISCUSSION TITLE
ci: Add dependabot for dependency management.

### DISCUSSION THREAD

**Discussion by afifurrohman-id** [tokio-rs/axum#2712] [Ideas]
It would be nice if dependabot can open PR for update dependencies for axum repository, such as using `cargo` or GitHub Actions too.

**Comment by jplatte** [comment]
What's the motivation for this?

**Reply by wangziling** [reply]
The [dependabot](https://github.com/apps/dependabot) can automatically check the dependencies' version daily or other durations.

E.g. https://github.com/salvo-rs/salvo/blob/main/.github/dependabot.yml
https://github.com/salvo-rs/salvo/pull/751#issue-2240749670

It can notice the project owner that some dependecies are upgraded for fixing/enhancing something.

Maybe this discussion initiator wanna this project can grasp the changing events of its related crates.😉

**Reply by jplatte** [reply]
Well, I think we're already generally very well aware of when dependencies can use upgrades. Actually doing the upgrades can sometimes take time, but not because we don't know that there is something to update :)
